cmd/xkcd-gtk/properties-dialog.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"github.com/gotk3/gotk3/gdk"
-	"github.com/gotk3/gotk3/gtk"
 	"log"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/gotk3/gotk3/gdk"
```

Comment:
Is separating url imports out a manual choice, or do newer version of go-fix do that?